docs/snakefiles/rules.rst

```diff
@@ -343,10 +343,18 @@ Resources are always meant to be specified as total per job, not by thread (i.e.
 In general, resources are just names to the Snakemake scheduler, i.e., Snakemake does not check whether a job exceeds a certain resource.
 However, resources are used to determine which jobs can be executed at a time while not exceeding the given limits at the command line.
 If no limits are given, the resources are ignored in local execution.
+
 In cluster or cloud execution, resources are always passed to the backend, even if ``--resources`` is not specified.
 Apart from making Snakemake aware of hybrid-computing architectures (e.g. with a limited number of additional devices like GPUs) this allows us to control scheduling in various ways, e.g. to limit IO-heavy jobs by assigning an artificial IO-resource to them and limiting it via the ``--resources`` flag.
 Resources must be ``int`` or ``str`` values. Note that you are free to choose any names for the given resources.
 
+When submitting :ref:`group jobs <job_grouping>` to the cluster, Snakemake calculates how many resources to request by first determining which component jobs can be run in parallel, and which must be run in series.
+For most resources, such as ``mem_mb`` or ``threads``, a sum will be taken across each parallel layer.
+The layer requiring the most resource (i.e. ``max()``) will determine the final amount requested.
+The only exception is ``runtime``.
+For it, ``max()`` will be used within each layer, then the total amount of time across all layers will be summed.
+If resource constraints are provided (via ``-resources`` or ``--cores``) Snakemake will prevent group jobs from requesting more than the constraint.
```

Comment:
Just caught this while reading through this stupendous PR.

```suggestion
If resource constraints are provided (via ``--resources`` or ``--cores``) Snakemake will prevent group jobs from requesting more than the constraint.
```

Reply:
Thanks for catching this!